### DISCUSSION TITLE
v form v-form fields didn't work with array

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [directus/directus#11472] [Archive]
**I'm trying to use the v-form component.**

fields | Array of fields to render. This can be used instead of the collection prop | undefined | Field[]

```
<v-form
    collection="articles"
    v-model="frmArticles"
    :fields="fieldsArticles"
/>
```

const fieldsArticles = ['title','text','date'];

### To Reproduce

I tried to limit displaying only some fields of the v-form, but even informing in the fields array. But it didn't work, it displays all the fields in the collection.

[https://github.com/directus/directus/wiki/v-form](https://github.com/directus/directus/wiki/v-form)

<details>
<summary>More Info</summary>

### What version of Directus are you using?

9.5.1

### What version of Node.js are you using?

v16.13.0

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying Directus?

Docker

</summary>

**Comment by azrikahar** [comment]
Moved this to discussion as it seems to be a usage issue rather than an actual bug within Directus.

---

Have you tried _removing_ the `collection` prop? The wiki you have linked does mention that 

> `fields` : Array of fields to render. This can be used instead of the collection prop

> Note: You have to pass either the collection or fields prop.

So since your intention is to use certain fields only, you should **remove** the `collection` prop or else it will display all fields in collection as you have mentioned.

Here's the underlying logic for this in `v-form`'s source code:

https://github.com/directus/directus/blob/b8c4317cb1cd9a8ed0eeb1ad4db35f8771588624/app/src/components/v-form/v-form.vue#L222-L232

**Comment by hermesalvesbr** [comment]
Hi,

I believe it's a bug, because I also passed without the collection and it didn't work either.

I informed only the fields of the collection articles and it was not. I also tried articles.title and it didn't work either.

`const fieldsIndvo = ref(["articles.title"]);`

Apparently it doesn't work or something is missing from the documentation.

**Reply by hermesalvesbr** [reply]
Sorry, i forgot :fields, i'm using just fields :( 

Everything it's ok! Thanks for your attention.

**Reply by azrikahar** [reply]
All good. Thanks for confirming! 😊
